<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TractionIsolate</class>
 <widget class="QWidget" name="TractionIsolate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}

QPushButton
{
	border-radius: 5px;
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
}</string>
  </property>
  <widget class="QPushButton" name="btn_mp1tcu1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp1 牵引TCU1隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp1tcu2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp1 牵引TCU2隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp1tcu3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp1 牵引TCU3隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp1tcu4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp1 牵引TCU4隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp2tcu1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp2 牵引TCU1隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp2tcu3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp2 牵引TCU3隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp2tcu4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp2 牵引TCU4隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mp2tcu2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Mp2 牵引TCU2隔离</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>520</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
